hazelcast-all/pom.xml

```diff
@@ -111,11 +111,19 @@
                                 </transformer>
                                 <transformer
                                         implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
+                                <transformer
+                                        implementation="org.apache.maven.plugins.shade.resource.ApacheNoticeResourceTransformer"/>
+                                <transformer
+                                        implementation="org.apache.maven.plugins.shade.resource.DontIncludeResourceTransformer">
+                                    <resource>com.hazelcast.org.codehaus.commons.compiler.properties</resource>
+                                    <resource>hazelcast-community-license.txt</resource>
```

Comment:
@jerrinot @Holmistr We do not include `hazelcast-community-license.txt` in `hazelcast-all.jar` at all. It is regarding below warning 
```
[WARNING] hazelcast-aws-3.3.jar, hazelcast-wm-4.0.jar define 1 overlapping resource: 
[WARNING]   - hazelcast-community-license.txt
```
I think we do not need to include it inside the jar. kindly advise.